doc/neps/nep-0023-restructuring-np-core.rst

```diff
@@ -0,0 +1,165 @@
+==============================================
+NEP 23 — Making details of ``np.core`` private
+==============================================
```

Comment:
Number needs updating to 46. Also needs a new file name.

Reply:
47 now